test/generateRoutes.test.js

```diff
@@ -19,7 +18,6 @@ describe('Herbs2Rest - Generate Routes', () => {
           }
         }
       }
-    }
   }
   const usecaseTest = () =>

```

Comment:
should have a test for this scenario.

In an ideal world, for every conditional you put in your code, you should have a test that positively tests that conditional and another that negatively tests that conditional.

So please add a test that validates and/or invalidates this "if(route.entity)"